opendkim/opendkim.8.in

```diff
@@ -262,6 +263,14 @@ The value may include two different canonicalizations separated by a
 slash ("/") character, in which case the first will be applied to the
 headers and the second to the body.
 .TP
+.I \-G
```

Comment:
Keep alphabetic order in the descriptions of command line options :)  This should be below of  `-g` option now.

Reply:
https://github.com/trusteddomainproject/OpenDKIM/pull/228/commits/ac1d2c6503109460a9c5ab9b07efc7942b82ac2e

Reply:
New place of `-G` is still above of `-g`.  It seems the order is lowercase first. (Other changs looks good to me, thank you!)

Reply:
https://github.com/trusteddomainproject/OpenDKIM/pull/228/commits/dfa3dfef5d657a3452b6a5c56c282ec8060b8247